os/tools/compression/Makefile

```diff
@@ -90,29 +90,25 @@ $(APPNAME): Makefile $(CONFIG) $(CCONFIG) $(COMPHEADER) $(OBJECTS)
 	@$(CC) $(LDFLAGS) $(OBJECTS) $(LIBFILES) -o $@
 
 $(CCONFIG): $(TINYARADIR)/include/tinyara/config.h
-	@echo "CP: config.h"
 	@cp $(TINYARADIR)/include/tinyara/config.h .
 
 # ==========================================================
 # Rule to retrieve compression header file
 # ==========================================================
 $(COMPHEADER): $(TINYARADIR)/include/tinyara/binfmt/compression/compression.h
-	@echo "CP: compression.h"
 	@cp $(TINYARADIR)/include/tinyara/binfmt/compression/compression.h .
 
 # =================================
 # Rule to retrieve the .config file
 # =================================
 $(CONFIG): $(TINYARADIR)/.config
-	@echo "CP: .config"
 	@cp $(TINYARADIR)/.config .
```

Comment:
As I left before, when we call `clean`, this is executed. Could you check the reason of calling this with clean?

Reply:
It is not calling copy now with clean.